### PR TITLE
shoe-312: made it christmasy

### PR DESCRIPTION
**Pull request by developerjaym**
Added background and text colors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Soul Shoes</title>
+    <style>
+        body {
+            background-color: red;
```
Comment (mlgvla) [inline index.html]: These colors might pose an accessibility problem for some end users

**Review by mlgvla — CHANGES_REQUESTED** [review]
Please change to a more accessible color scheme.

File: index.html (hunk 1 above)
Comment (developerjaym) [reply index.html]: Whatever

**Comment by developerjaym** [comment]
> background-color: red;

FIXED!

**Review by mlgvla — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (developerjaym) [reply index.html]: Consider instead
```
background-color: pink
```

**Comment by developerjaym** [comment]
This doesn't meet the 3 tasks needed for this user story to be complete.

**Pull request merged**